<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1265</width>
    <height>848</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>662</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>6000</width>
    <height>6000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1241</width>
      <height>821</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1241</width>
       <height>759</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Business Page</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>171</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>80</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>10</y>
         <width>191</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;
background-color: rgb(181, 206, 227);</string>
       </property>
       <property name="title">
        <string>Zipcode</string>
       </property>
       <widget class="QTableWidget" name="Zipcodelist">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>991</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;
</string>
      </property>
      <property name="title">
       <string>                       Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>951</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>0</y>
        <width>771</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;
background-color: rgb(181, 206, 227);</string>
      </property>
      <property name="title">
       <string>        Zipcode Statistics </string>
      </property>
      <widget class="QListWidget" name="numofbusinesses">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string># of Businesses</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>10</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Top Categories</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="topCategories">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>40</y>
         <width>331</width>
         <height>161</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Total Population</string>
       </property>
      </widget>
      <widget class="QListWidget" name="totalpopulation">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Average Income</string>
       </property>
      </widget>
      <widget class="QListWidget" name="avgincome">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>150</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>251</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
      </property>
      <property name="title">
       <string>Category Filter</string>
      </property>
      <widget class="QListWidget" name="categorylist">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>221</width>
         <height>281</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>430</y>
        <width>991</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(181, 206, 227);
font: 10pt &quot;Lucida Sans Unicode&quot;;
</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTableWidget" name="popBusinesses">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>451</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Popular Businesses (in zipcode)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>10</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans Unicode&quot;;</string>
       </property>
       <property name="text">
        <string>Successful Businesses (in zipcode)</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="successfulbusiness">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>40</y>
         <width>501</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1241</width>
       <height>759</height>
      </rect>
     </property>
     <attribute name="label">
      <string>User Page</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
